configure.ac

```diff
@@ -598,6 +598,12 @@ MOTR_SEARCH_LIBS([io_getevents], [aio c], [AIO_LIBS],
 )
 AC_SUBST([AIO_LIBS])
 
+# check for libedit library
+MOTR_SEARCH_LIBS([readline], [c edit], [LIBEDIT_LIBS],
```

Comment:
readline is still mentioned here.

Reply:
Yes, it's a function call to check - readline(). The API in libedit is compatible with readline lib.